auslib/db.py

```diff
@@ -2064,27 +2061,20 @@ def update(self, where, what, changed_by, old_data_version, transaction=None, dr
                     ancestor_blob = ancestor_change.get('data')
                     tip_release = self.getReleases(name=name, transaction=transaction)[0]
                     tip_blob = tip_release.get('data')
-                    m = dictdiffer.merge.Merger(ancestor_blob, tip_blob, blob, {})
                     try:
-                        m.run()
-                        # Merger merges the patches into a single unified patch,
-                        # but we need dictdiffer.patch to actually apply the patch
-                        # to the original blob
-                        unified_blob = dictdiffer.patch(m.unified_patches, ancestor_blob)
-                        # converting the resultant dict into a blob and then
-                        # converting it to JSON
-                        what['data'] = unified_blob
-                        # we want the data_version for the dictdiffer.merged blob to be one
-                        # more than that of the latest blob
-                        tip_data_version = tip_release['data_version']
-                        super(Releases, self).update(where={"name": name}, what=what, changed_by=changed_by, old_data_version=tip_data_version,
-                                                     transaction=transaction, dryrun=dryrun)
-                        # cache will have a data_version of one plus the tip
-                        # data_version
-                        new_data_version = tip_data_version + 1
-                    except dictdiffer.merge.UnresolvedConflictsException:
-                        self.log.debug("latest version of release %s cannot be merged with new blob" % name)
+                        what['data'] = createBlob(merge_dicts(ancestor_blob, tip_blob, blob))
+                    except ValueError:
+                        self.log.exception("Couldn't merge blobs.")
```

Comment:
How merge_dicts raises `ValueError("Cannot merge blobs: left and right are both changing '{}'".format(key.encode('ascii', 'replace')))`, here can log the message.

Reply:
I'm not exactly sure what you're suggesting here - can you clarify? If you're saying it should log the message from merge_dicts, that's already happening by log.exception() (it includes the traceback + message).

Reply:
Ah OK, that's it. I didn't know.